HTML_PAGES/company.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Company Page</title>
+    <link rel="icon" href="/pics/favicon.png" type="image/png" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x"
+      crossorigin="anonymous"
+    />
+    <link href="/CSS/style.css" rel="stylesheet" />
+  </head>
+  <body>
+    <div class="container">
+      <div id="company-data"></div>
+      <canvas id="bar-chart" width="800" height="450"> </canvas>
+    </div>
+
+    <script src="/JS/constVars.js"></script>
+    <script src="/JS/SearchResult.js"></script>
```

Comment:
the searchResults class should be used for something else. Please see ITC solution for the project when published.